### PR TITLE
Remove SEL MACRO

### PR DESCRIPTION
**Pull request by tkcjesse**
I never claimed to be perfect... let me know if there's something messed
over. A few look overs would be much appreciated. Just did this after a
12hr work day. Searched all the files 3x after completion and couldn't
find anymore of "SEL"... So pretty sure I got them all. Also looked over
the commit, looks good.

Also for those who are like "OMG wtf?! +329 −332" <- It's the removal of the macro from the macro file.. Calm down. @BoGuu 

Credits to that @BoGuu  guy too for helping out on some questions and teaching a few things.

### CODE REVIEW

File: Altis_Life.Altis/core/housing/fn_raidHouse.sqf [hunk 1]
```diff
@@ -51,12 +51,12 @@ if (player distance _house > 13) exitWith {life_action_inUse = false; titleText[
...
 
-_houseInvData = SEL(_houseInv,0);
-_houseInvVal = SEL(_houseInv,1);
+_houseInvData = (_houseInv select 0);
+_houseInvVal = (_houseInv select 1);
 _value = 0;
 {
-    _var = SEL(_x,0);
-    _val = SEL(_x,1);
+    _var = (x select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/housing/fn_raidHouse.sqf]: Marking

File: Altis_Life.Altis/core/functions/fn_loadDeadGear.sqf [hunk 2]
```diff
@@ -9,22 +9,22 @@
...
-_uitems = SEL(_loadout,11);
-_vitems = SEL(_loadout,12);
-_bitems = SEL(_loadout,13);
-_headgear = SEL(_loadout,14);
-_goggles = SEL(_loadout,15);
+_primary = (_loadout select 0);
+_launcher = (_loadout select 1);
+_handgun = (_loadout select 2);
+_magazines = (_loadout select 3);
+_uniform = (_loadout select 4);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_loadDeadGear.sqf]: Ehhhhhhhhhh, params

File: Altis_Life.Altis/core/shops/fn_vehicleShopMenu.sqf [hunk 3]
```diff
@@ -46,11 +46,11 @@ ctrlShow [2304,false];
 
 //Loop through
 {
-    _className = SEL(_x,0);
-    _levelAssert = SEL(_x,1);
-    _levelName = SEL(_levelAssert,0);
-    _levelType = SEL(_levelAssert,1);
-    _levelValue = SEL(_levelAssert,2);
+    _className = (x select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_vehicleShopMenu.sqf]: !!!!!!!!!!!!!!!!!!!!!!!!!!

File: Altis_Life.Altis/core/shops/fn_vehicleShopLBChange.sqf [hunk 4]
```diff
@@ -76,9 +76,9 @@ if (!isClass (missionConfigFile >> "LifeCfgVehicles" >> _classNameLife)) then {
 _colorArray = M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures");
 
 {
-    _flag = SEL(_x,1);
-    _textureName = SEL(_x,0);
-    if (SEL(life_veh_shop,2) isEqualTo _flag) then {
+    _flag = (x select 1);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_vehicleShopLBChange.sqf]: !!!!!!!!!!!!!!!!!!!!!!!!!

File: Altis_Life.Altis/core/shops/fn_virt_menu.sqf [hunk 5]
```diff
@@ -7,17 +7,17 @@
...
-_levelValue = SEL(_levelAssert,2);
-_levelMsg = SEL(_levelAssert,3);
-life_shop_type = SEL(_this,3);
-life_shop_npc = SEL(_this,0);
+_shopSide = M_CONFIG(getText,"VirtualShops",(_this select 3),"side");
+_license = M_CONFIG(getText,"VirtualShops",(_this select 3),"license");
+_levelAssert = M_CONFIG(getArray,"VirtualShops",(_this select 3),"level");
+_levelName = (_levelAssert select 0);
+_levelType = (_levelAssert select 1);
+_levelValue = (_levelAssert select 2);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_virt_menu.sqf]: Params

File: Altis_Life.Altis/core/shops/fn_weaponShopFilter.sqf [hunk 6]
```diff
@@ -23,14 +23,14 @@ if ((uiNamespace getVariable ["Weapon_Magazine",0]) isEqualTo 1 || (uiNamespace
     if ((uiNamespace getVariable ["Weapon_Magazine",0]) isEqualTo 1) then {
         _config = M_CONFIG(getArray,"WeaponShops",_shop,"mags");
         {
-            _var = SEL(_x,0);
+            _var = (x select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_weaponShopFilter.sqf]: !!!!!!!!!!!!!!

File: Altis_Life.Altis/core/shops/fn_weaponShopMenu.sqf [hunk 7]
```diff
@@ -8,14 +8,14 @@
...
-_levelAssert = M_CONFIG(getArray,"WeaponShops",(SEL(_this,3)),"level");
-_levelName = SEL(_levelAssert,0);
-_levelType = SEL(_levelAssert,1);
-_levelValue = SEL(_levelAssert,2);
-_levelMsg = SEL(_levelAssert,3);
+_shopTitle = M_CONFIG(getText,"WeaponShops",(_this select 3),"name");
+_shopSide = M_CONFIG(getText,"WeaponShops",(_this select 3),"side");
+_license = M_CONFIG(getText,"WeaponShops",(_this select 3),"license");
+_levelAssert = M_CONFIG(getArray,"WeaponShops",(_this select 3),"level");
+_levelName = (_levelAssert select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_weaponShopMenu.sqf]: Params

File: Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf [hunk 8]
```diff
@@ -37,7 +37,7 @@ if ((uiNamespace getVariable ["Weapon_Shop_Filter",0]) isEqualTo 1) then {
                     _itemArray = FETCH_CONFIG2(getArray,"CfgWeapons",_item,"magazines");
                     _bool = false;
                     {
-                        _var = SEL(_x,0);
+                        _var = (x select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf]: !!!!!!!!!!!!!!!!!!!!

File: Altis_Life.Altis/core/vehicle/fn_vehInventory.sqf [hunk 9]
```diff
@@ -39,13 +39,13 @@ if (count _data isEqualTo 0) then {_veh setVariable["Trunk",[[],0],true]; _data
 
 //Trunk Inventory Items
 {
-    _name = M_CONFIG(getText,"VirtualItems",SEL(_x,0),"displayName");
-    _val = SEL(_x,1);
+    _name = M_CONFIG(getText,"VirtualItems",(x select 0),"displayName");
+    _val = (x select 1);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/vehicle/fn_vehInventory.sqf]: !!!!!!!!!!!!!!!!!!!!!!!!

File: Altis_Life.Altis/dialog/function/fn_impoundMenu.sqf [hunk 10]
```diff
@@ -23,13 +23,13 @@ _control = CONTROL(2800,2802);
 lbClear _control;
 
 {
-    _vehicleInfo = [SEL(_x,2)] call life_fnc_fetchVehInfo;
-    _control lbAdd (SEL(_vehicleInfo,3));
-    _tmp = [SEL(_x,2),SEL(_x,8)];
+    _vehicleInfo = [(x select 2)] call life_fnc_fetchVehInfo;
+    _control lbAdd (_vehicleInfo select 3);
+    _tmp = [(x select 2),(_x select 8)];
```
Comment (BoGuu) [inline Altis_Life.Altis/dialog/function/fn_impoundMenu.sqf]: !!!!!!!!!!!!!!!!!!!!!!!!

File: Altis_Life.Altis/core/pmenu/fn_wantedMenu.sqf [hunk 11]
```diff
@@ -33,17 +33,16 @@ lbClear _list2; //Purge the list
...
+    _list2 lbAdd format["%1 - $%2 (%3)",_x select 0,_x select 1,_x select 2];
   };
-    _list2 lbSetData [(lbSize _list2)-1,(_x select 2)];
+    _list2 lbSetData [(lbSize _list2)-1,_x select 2];
 } forEach _crimes;
 
 ctrlSetText[2404,"Establishing connection..."];
 
-if (FETCH_CONST(life_coplevel) < 3 && FETCH_CONST(life_adminlevel) isEqualTo 0) then
-{
```
Comment (tkcjesse) [inline Altis_Life.Altis/core/pmenu/fn_wantedMenu.sqf]: Here's the +298 −299

**Comment by tkcjesse** [comment]
https://github.com/ArmaLife/Framework/issues/664

**Comment by tkcjesse** [comment]
I'll catch up on the params at a later time. Gotta get food and hit the hay.

**Comment by Jawshy** [comment]
@tkcjesse 
The following is written to the local/client.rpt file upon accessing a vehicle shop: 
```
16:08:25 Error in expression <e] call life_fnc_fetchVehInfo;
_control lbAdd _vehicleInfo select 3;
_control lb>
16:08:25   Error position: <lbAdd _vehicleInfo select 3;
_control lb>
16:08:25   Error lbadd: Type Array, expected String
16:08:25 File mpmissions\__CUR_MP.Altis\core\shops\fn_vehicleShopMenu.sqf, line 70
```

Config_Master.hpp: 
```hpp
    vehicleShop_3D = true; //Add preview 3D inside Shop vehicle.       Default : False
```

As of https://github.com/ArmaLife/Framework/pull/749/commits/ba93f011223c0a1b712ca1b37042b0f5639da357.

**Comment by tkcjesse** [comment]
Can you try...   lbAdd (_vehicleInfo select 3); in that file? I think it may be in two spots there. I'm not sure what the issue is. I will have to look at the file in about 18hrs.

**Comment by Jawshy** [comment]
@tkcjesse 
> Can you try... lbAdd (_vehicleInfo select 3); in that file? [...] 

fn_vehicleShopMenu.sqf, line 70: 
from `_control lbAdd _vehicleInfo select 3;` 
to `_control lbAdd (_vehicleInfo select 3);`
Works.

**Comment by Jawshy** [comment]
The following is written to the server.rpt file upon selecting "Get ID" for a player in the admin menu: 
```
2016/05/26, 18:54:39 Error in expression <"









private["_id","_ret"];
_id = owner _this select 0;
_ret = owner _this>
2016/05/26, 18:54:39   Error position: <owner _this select 0;
_ret = owner _this>
2016/05/26, 18:54:39   Error owner: Type Array, expected Object
2016/05/26, 18:54:39 File life_server\Functions\Systems\fn_getID.sqf, line 11
```

As of https://github.com/ArmaLife/Framework/pull/749/commits/ba93f011223c0a1b712ca1b37042b0f5639da357.

**Comment by tkcjesse** [comment]
Yeah... I'll go back through and readd some quotes for stuff when i get home.

**Comment by BoGuu** [comment]
http://i.imgur.com/meftV0q.gif

**Comment by tkcjesse** [comment]
ANNNNNDDDD should be good, if not good more than good since I added some extra in places where they probably weren't needed. I just don't have time till the weekend or possibly a bit into the start of next week to further refine. If anyone notices/finds anything feel free to leeett me know.

**Comment by Jawshy** [comment]
@tkcjesse, the following is written to the server.rpt when retrieving a vehicle from a garage:
```
2016/05/27, 13:13:46 Error in expression <cle_fuel") isEqualTo 1) then {
_vehicle setFuel _vInfo select 11;
}else{
_vehicl>
2016/05/27, 13:13:46   Error position: <setFuel _vInfo select 11;
}else{
_vehicl>
2016/05/27, 13:13:46   Error setfuel: Type Array, expected Number
2016/05/27, 13:13:46 File life_server\Functions\Systems\fn_spawnVehicle.sqf, line 126
```
As of commit https://github.com/ArmaLife/Framework/pull/749/commits/ba47925a32c9b987912e7442692ca1507be07f70.

**Comment by Jawshy** [comment]
I think this is ready to merge, @tkcjesse. I'm struggling to find any more issues.

**Comment by tkcjesse** [comment]
Fantastic. Yeah.  There may be one hiding but it can easily get wrapped in parens if need be.  Thanks for the help jawsh.

**Comment by tkcjesse** [comment]
What do we think Ladies? @BoGuu

File: Altis_Life.Altis/core/actions/fn_pickupMoney.sqf [hunk 12]
```diff
@@ -10,7 +10,7 @@ private "_value";
 if ((time - life_action_delay) < 1.5) exitWith {hint localize "STR_NOTF_ActionDelay"; _this setVariable ["inUse",false,true];};
 if (isNull _this || {player distance _this > 3}) exitWith {_this setVariable ["inUse",false,true];};
 
-_value = SEL((_this getVariable "item"),1);
+_value = ((_this getVariable "item") select 1);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/actions/fn_pickupMoney.sqf]: Reference

File: Altis_Life.Altis/core/actions/fn_processAction.sqf [hunk 13]
```diff
@@ -29,10 +29,10 @@ _itemInfo = [_materialsRequired,_materialsGiven,_noLicenseCost,(localize format[
 if (count _itemInfo isEqualTo 0) exitWith {life_action_inUse = false;};
 
 //Setup vars.
-_oldItem = SEL(_itemInfo,0);
-_newItem = SEL(_itemInfo,1);
-_cost = SEL(_itemInfo,2);
-_upp = SEL(_itemInfo,3);
+_oldItem = _itemInfo select 0;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/actions/fn_processAction.sqf]: Reference

File: Altis_Life.Altis/core/actions/fn_storeVehicle.sqf [hunk 14]
```diff
@@ -10,13 +10,13 @@ private["_nearVehicles","_vehicle"];
...
 } else {
-    _nearVehicles = nearestObjects[getPos SEL(_this ,0),["Car","Air","Ship"],30]; //Fetch vehicles within 30m.
+    _nearVehicles = nearestObjects[getPos (_this select 0),["Car","Air","Ship"],30]; //Fetch vehicles within 30m.
     if (count _nearVehicles > 0) then {
         {
             if (!isNil "_vehicle") exitWith {}; //Kill the loop.
             _vehData = _x getVariable ["vehicle_info_owners",[]];
             if (count _vehData  > 0) then {
-                _vehOwner = SEL(SEL(_vehData,0),0);
+                _vehOwner = ((_vehData select 0) select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/actions/fn_storeVehicle.sqf]: Reference

File: Altis_Life.Altis/core/functions/fn_giveDiff.sqf [hunk 15]
```diff
@@ -7,11 +7,11 @@
     ??A?SD?ADS?A
 */
 private["_unit","_item","_val","_from","_bool"];
-_unit = SEL(_this,0);
+_unit = _this select 0;
 if (_unit != player) exitWith {};
-_item = SEL(_this,1);
-_val = SEL(_this,2);
-_from = SEL(_this,3);
+_item = _this select 1;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_giveDiff.sqf]: Reference

File: Altis_Life.Altis/core/functions/fn_handleDamage.sqf [hunk 16]
```diff
@@ -7,11 +7,11 @@
     Handles damage, specifically for handling the 'tazer' pistol and nothing else.
 */
 private["_unit","_damage","_source","_projectile","_part","_curWep"];
-_unit = SEL(_this,0);
-_part = SEL(_this,1);
-_damage = SEL(_this,2);
-_source = SEL(_this,3);
-_projectile = SEL(_this,4);
+_unit = _this select 0;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_handleDamage.sqf]: Reference

File: Altis_Life.Altis/core/functions/fn_keyHandler.sqf [hunk 17]
```diff
@@ -7,16 +7,16 @@
 *    Main key handler for event 'keyDown'
 */
 private ["_handled","_shift","_alt","_code","_ctrl","_alt","_ctrlKey","_veh","_locked","_interactionKey","_mapKey","_interruptionKeys"];
-_ctrl = SEL(_this,0);
-_code = SEL(_this,1);
-_shift = SEL(_this,2);
-_ctrlKey = SEL(_this,3);
-_alt = SEL(_this,4);
+_ctrl = _this select 0;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: Reference

File: Altis_Life.Altis/core/functions/fn_loadDeadGear.sqf [hunk 18]
```diff
@@ -9,22 +9,22 @@
...
-_items = SEL(_loadout,7);
-_primitems = SEL(_loadout,8);
-_secitems = SEL(_loadout,9);
-_handgunitems = SEL(_loadout,10);
-_uitems = SEL(_loadout,11);
-_vitems = SEL(_loadout,12);
-_bitems = SEL(_loadout,13);
-_headgear = SEL(_loadout,14);
-_goggles = SEL(_loadout,15);
+_primary = _loadout select 0;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_loadDeadGear.sqf]: ReferenceReferenceReferenceReferenceReference

File: Altis_Life.Altis/core/functions/fn_receiveItem.sqf [hunk 19]
```diff
@@ -7,11 +7,11 @@
     Receive an item from a player.
 */
 private["_unit","_val","_item","_from","_diff"];
-_unit = SEL(_this,0);
+_unit = _this select 0;
 if (_unit != player) exitWith {};
-_val = SEL(_this,1);
-_item = SEL(_this,2);
-_from = SEL(_this,3);
+_val = _this select 1;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_receiveItem.sqf]: Reference

File: Altis_Life.Altis/core/housing/fn_raidHouse.sqf [hunk 20]
```diff
@@ -51,12 +51,12 @@ if (player distance _house > 13) exitWith {life_action_inUse = false; titleText[
 if (!alive player) exitWith {life_action_inUse = false;};
 life_action_inUse = false;
 
-_houseInvData = SEL(_houseInv,0);
-_houseInvVal = SEL(_houseInv,1);
+_houseInvData = (_houseInv select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/housing/fn_raidHouse.sqf]: ................. Reference

File: Altis_Life.Altis/core/pmenu/fn_s_onChar.sqf [hunk 21]
```diff
@@ -14,10 +14,10 @@
         2: STRING (Case option)
 */
 private["_control","_code","_slider","_value","_varName","_onKeyUp"];
-_control = SEL(_this,0);
-_code = SEL(_this,1);
-_slider = SEL(_this,2);
-_onKeyUp = SEL(_this,3);
+_control = _this select 0;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/pmenu/fn_s_onChar.sqf]: Reference

File: Altis_Life.Altis/core/session/fn_requestReceived.sqf [hunk 22]
```diff
@@ -32,82 +32,82 @@ if (!isServer && (!isNil "life_adminlevel" || !isNil "life_coplevel" || !isNil "
...
-        CONST(life_coplevel, parseNumber(SEL(_this,7)));
+        CONST(life_coplevel, parseNumber(_this select 7));
         CONST(life_medicLevel,0);
-        life_blacklisted = SEL(_this,9);
+        life_blacklisted = _this select 9;
         if (LIFE_SETTINGS(getNumber,"save_playerStats") isEqualTo 1) then {
-            life_hunger = SEL(SEL(_this,10),0);
-            life_thirst = SEL(SEL(_this,10),1);
-            player setDamage SEL(SEL(_this,10),2);
+            life_hunger = ((_this select 10) select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/session/fn_requestReceived.sqf]: Reference

File: Altis_Life.Altis/core/session/fn_requestReceived.sqf [hunk 23]
```diff
@@ -32,82 +32,82 @@ if (!isServer && (!isNil "life_adminlevel" || !isNil "life_coplevel" || !isNil "
...
 
     case independent: {
-        CONST(life_medicLevel, parseNumber(SEL(_this,7)));
+        CONST(life_medicLevel, parseNumber(_this select 7));
         CONST(life_coplevel,0);
         if (LIFE_SETTINGS(getNumber,"save_playerStats") isEqualTo 1) then {
-            life_hunger = SEL(SEL(_this,9),0);
-            life_thirst = SEL(SEL(_this,9),1);
-            player setDamage SEL(SEL(_this,9),2);
+            life_hunger = ((_this select 9) select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/session/fn_requestReceived.sqf]: Reference

File: Altis_Life.Altis/core/shops/fn_FuelLBchange.sqf [hunk 24]
```diff
@@ -18,7 +18,7 @@ _vehicle = (vehiclefuelList select _vindex) select 0;
 _vehicleInfo = [_className] call life_fnc_fetchVehInfo;
 
 _fuel = fuel _vehicle;
-_fueltank = SEL(_vehicleInfo,12);
+_fueltank = (_vehicleInfo select 12);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_FuelLBchange.sqf]: Reference

File: Altis_Life.Altis/core/shops/fn_chopShopSell.sqf [hunk 25]
```diff
@@ -14,7 +14,7 @@ _vehicle = _control lbData (lbCurSel _control);
 _vehicle = call compile format["%1", _vehicle];
 _chopable = LIFE_SETTINGS(getArray,"chopShop_vehicles");
 _nearVehicles = nearestObjects [getMarkerPos life_chopShop,_chopable,25];
-_vehicle = SEL(_nearVehicles,_vehicle);
+_vehicle = (_nearVehicles select _vehicle);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_chopShopSell.sqf]: Reference

File: Altis_Life.Altis/core/shops/fn_clothingFilter.sqf [hunk 26]
```diff
@@ -53,17 +53,17 @@ _array = switch (_selection) do {
...
 
 {
-    _className = SEL(_x,0);
-    _displayName = SEL(_x,1);
-    _price = SEL(_x,2);
-    _levelAssert = SEL(_x,3);
-    _levelName = SEL(_levelAssert,0);
-    _levelType = SEL(_levelAssert,1);
-    _levelValue = SEL(_levelAssert,2);
+    _className = (_x select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_clothingFilter.sqf]: ReferenceReferenceReference

File: Altis_Life.Altis/core/shops/fn_vehicleShopLBChange.sqf [hunk 27]
```diff
@@ -39,11 +39,11 @@ switch (playerSide) do {
 
 _vehicleInfo = [_className] call life_fnc_fetchVehInfo;
 _trunkSpace = [_className] call life_fnc_vehicleWeightCfg;
-_maxspeed = _vehicleInfo select 8;
-_horsepower = _vehicleInfo select 11;
-_passengerseats = _vehicleInfo select 10;
-_fuel = _vehicleInfo select 12;
-_armor = _vehicleInfo select 9;
+_maxspeed = (_vehicleInfo select 8);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_vehicleShopLBChange.sqf]: ReferenceReferenceReference

File: Altis_Life.Altis/core/shops/fn_vehicleShopMenu.sqf [hunk 28]
```diff
@@ -46,11 +46,11 @@ ctrlShow [2304,false];
 
 //Loop through
 {
-    _className = SEL(_x,0);
-    _levelAssert = SEL(_x,1);
-    _levelName = SEL(_levelAssert,0);
-    _levelType = SEL(_levelAssert,1);
-    _levelValue = SEL(_levelAssert,2);
+    _className = _x select 0;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_vehicleShopMenu.sqf]: ReferenceReference

File: Altis_Life.Altis/core/shops/fn_virt_menu.sqf [hunk 29]
```diff
@@ -7,17 +7,17 @@
...
+_levelMsg = (_levelAssert select 3);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_virt_menu.sqf]: Reference

File: Altis_Life.Altis/core/shops/fn_virt_sell.sqf [hunk 30]
```diff
@@ -32,7 +32,7 @@ if (life_shop_type isEqualTo "drugdealer") then {
     _array = life_shop_npc getVariable["sellers",[]];
     _ind = [getPlayerUID player,_array] call TON_fnc_index;
     if (!(_ind isEqualTo -1)) then {
-        _val = SEL(SEL(_array,_ind),2);
+        _val = ((_array select _ind) select 2);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_virt_sell.sqf]: Reference

File: Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf [hunk 31]
```diff
@@ -19,7 +19,7 @@ if ((uiNamespace getVariable ["Weapon_Shop_Filter",0]) isEqualTo 1) then {
     _item = CONTROL_DATAI(_control,_index);
     _itemArray = M_CONFIG(getArray,"WeaponShops",_shop,"items");
     _item = [_item,_itemArray] call TON_fnc_index;
-    _price = SEL(SEL(_itemArray,_item),3);
+    _price = ((_itemArray select _item) select 3);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf]: Reference

File: Altis_Life.Altis/core/vehicle/fn_colorVehicle.sqf [hunk 32]
```diff
@@ -34,7 +34,7 @@ if (!isClass (missionConfigFile >> "LifeCfgVehicles" >> _classNameLife)) then {
     diag_log format["%1: LifeCfgVehicles class doesn't exist",_className];
 };
 
-_textures = SEL(SEL(M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures"),_index),2);
+_textures = ((M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures") select _index) select 2);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/vehicle/fn_colorVehicle.sqf]: Reference

File: Altis_Life.Altis/core/vehicle/fn_deviceMine.sqf [hunk 33]
```diff
@@ -124,12 +124,12 @@ for "_i" from 0 to 1 step 0 do {
     };
 
     _vehicle_data = _vehicle getVariable ["Trunk",[[],0]];
-    _inv = _vehicle_data select 0;
-    _space = _vehicle_data select 1;
+    _inv = (_vehicle_data select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/vehicle/fn_deviceMine.sqf]: Reference

File: Altis_Life.Altis/core/vehicle/fn_vehInventory.sqf [hunk 34]
```diff
@@ -18,11 +18,11 @@ lbClear _pInv;
...
 _veh_data = [_veh] call life_fnc_vehicleWeight;
 
-if (SEL(_veh_data,0) isEqualTo -1) exitWith {closeDialog 0};
+if ((_veh_data select 0) isEqualTo -1) exitWith {closeDialog 0};
 
-ctrlSetText[3504,format["Weight: %1/%2",_veh_data select 1,_veh_data select 0]];
+ctrlSetText[3504,format["Weight: %1/%2",(_veh_data select 1),(_veh_data select 0)]];
 _data = _veh getVariable ["Trunk",[]];
-if (count _data isEqualTo 0) then {_veh setVariable["Trunk",[[],0],true]; _data = [];} else {_data = _data select 0;};
+if (count _data isEqualTo 0) then {_veh setVariable["Trunk",[[],0],true]; _data = [];} else {_data = (_data select 0);};
```
Comment (BoGuu) [inline Altis_Life.Altis/core/vehicle/fn_vehInventory.sqf]: Reference

File: Altis_Life.Altis/dialog/function/fn_garageLBChange.sqf [hunk 35]
```diff
@@ -8,21 +8,21 @@
...
+_className = (_dataArr select 0);
 _classNameLife = _className;
 
 if (!isClass (missionConfigFile >> "LifeCfgVehicles" >> _classNameLife)) then {
     _classNameLife = "Default"; //Use Default class if it doesn't exist
     diag_log format["%1: LifeCfgVehicles class doesn't exist",_className];
 };
 
-_vehicleColor = SEL(SEL(M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures"),SEL(_dataArr,1)),0);
+_vehicleColor = ((M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures") select (_dataArr select 1)) select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/dialog/function/fn_garageLBChange.sqf]: Reference

File: Altis_Life.Altis/dialog/function/fn_spawnPointCfg.sqf [hunk 36]
```diff
@@ -28,14 +28,14 @@ for "_i" from 0 to count(_spawnCfg)-1 do {
         _curConfig = (_spawnCfg select _i);
     _licenses = getArray(_curConfig >> "licenses");
     _level = getArray(_curConfig >> "level");
-        _levelName = SEL(_level,0);
-        _levelType = SEL(_level,1);
-        _levelValue = SEL(_level,2);
+        _levelName = (_level select 0);
```
Comment (BoGuu) [inline Altis_Life.Altis/dialog/function/fn_spawnPointCfg.sqf]: Reference

File: life_hc/MySQL/Vehicles/fn_chopShopSell.sqf [hunk 37]
```diff
@@ -26,8 +26,8 @@ _displayName = FETCH_CONFIG2(getText,"CfgVehicles",typeOf _vehicle, "displayName
 
 _dbInfo = _vehicle getVariable ["dbInfo",[]];
 if (count _dbInfo > 0) then {
-    _uid = SEL(_dbInfo,0);
-    _plate = SEL(_dbInfo,1);
+    _uid = (_dbInfo select 0);
```
Comment (BoGuu) [inline life_hc/MySQL/Vehicles/fn_chopShopSell.sqf]: Reference

File: life_hc/MySQL/Vehicles/fn_spawnVehicle.sqf [hunk 38]
```diff
@@ -72,10 +72,10 @@ if (count _nearVehicles > 0) exitWith {
 
 _query = format["UPDATE vehicles SET active='1', damage='""[]""' WHERE pid='%1' AND id='%2'",_pid,_vid];
 
-_trunk = [_vInfo select 9] call HC_fnc_mresToArray;
-_gear = [_vInfo select 10] call HC_fnc_mresToArray;
-_damage = [_vInfo select 12] call HC_fnc_mresToArray;
-_wasIllegal = _vInfo select 13;
+_trunk = [(_vInfo select 9)] call HC_fnc_mresToArray;
```
Comment (BoGuu) [inline life_hc/MySQL/Vehicles/fn_spawnVehicle.sqf]: Reference

File: life_hc/MySQL/Vehicles/fn_spawnVehicle.sqf [hunk 39]
```diff
@@ -159,20 +159,20 @@ if (count _damage > 0 && (LIFE_SETTINGS(getNumber,"save_vehicle_damage") isEqual
...
 
     for "_i" from 0 to ((count _damage) - 1) do {
-        _vehicle setHitPointDamage [format["%1",((_parts select 0) select _i)],(_damage select _i)];
+        _vehicle setHitPointDamage [format["%1",((_parts select 0) select _i)],_damage select _i];
     };
 };
 
 //Sets of animations
-if (SEL(_vInfo,1) isEqualTo "civ" && SEL(_vInfo,2) isEqualTo "B_Heli_Light_01_F" && !(SEL(_vInfo,8) isEqualTo 13)) then {
+if ((_vInfo select 1) isEqualTo "civ" && ((_vInfo select 2)) isEqualTo "B_Heli_Light_01_F" && !((_vInfo select 8) isEqualTo 13)) then {
```
Comment (BoGuu) [inline life_hc/MySQL/Vehicles/fn_spawnVehicle.sqf]: Reference

File: life_server/Functions/MySQL/fn_asyncCall.sqf [hunk 40]
```diff
@@ -21,7 +21,7 @@ _key = EXTDB format["%1:%2:%3",_mode,FETCH_CONST(life_sql_id),_queryStmt];
 if (_mode isEqualTo 1) exitWith {true};
 
 _key = call compile format["%1",_key];
-_key = SEL(_key,1);
+_key = (_key select 1);
```
Comment (BoGuu) [inline life_server/Functions/MySQL/fn_asyncCall.sqf]: Reference

File: life_server/Functions/Systems/fn_chopShopSell.sqf [hunk 41]
```diff
@@ -23,8 +23,8 @@ _unit = owner _unit;
 
 _dbInfo = _vehicle getVariable ["dbInfo",[]];
 if (count _dbInfo > 0) then {
-    _uid = SEL(_dbInfo,0);
-    _plate = SEL(_dbInfo,1);
+    _uid = (_dbInfo select 0);
```
Comment (BoGuu) [inline life_server/Functions/Systems/fn_chopShopSell.sqf]: Reference

File: life_server/Functions/Systems/fn_clientDisconnect.sqf [hunk 42]
```diff
@@ -8,11 +8,11 @@
     clean up their storage boxes in their house. Also, saves player infos & position.
 */
 private["_unit","_id","_uid","_name","_alive","_position","_side"];
-_unit = SEL(_this,0);
+_unit = _this select 0;
 if (isNull _unit) exitWith {};
-_id = SEL(_this,1);
-_uid = SEL(_this,2);
-_name = SEL(_this,3);
+_id = _this select 1;
```
Comment (BoGuu) [inline life_server/Functions/Systems/fn_clientDisconnect.sqf]: Reference

File: life_server/Functions/Systems/fn_spawnVehicle.sqf [hunk 43]
```diff
@@ -68,10 +68,10 @@ if (count _nearVehicles > 0) exitWith {
 
 _query = format["UPDATE vehicles SET active='1', damage='""[]""' WHERE pid='%1' AND id='%2'",_pid,_vid];
 
-_trunk = [_vInfo select 9] call DB_fnc_mresToArray;
-_gear = [_vInfo select 10] call DB_fnc_mresToArray;
-_damage = [_vInfo select 12] call DB_fnc_mresToArray;
-_wasIllegal = _vInfo select 13;
+_trunk = [(_vInfo select 9)] call DB_fnc_mresToArray;
```
Comment (BoGuu) [inline life_server/Functions/Systems/fn_spawnVehicle.sqf]: Reference

**Comment by BoGuu** [comment]
:+1:

**Comment by tkcjesse** [comment]
Man, so glad I had clicked "Squash and Merge"... man I really hate my life. I'm leaving. I'm a disappoint to even myself.

**Pull request merged**